src/main/java/graphql/execution/ConditionalNodes.java

```diff
@@ -9,7 +9,6 @@
 
 import static graphql.Directives.IncludeDirective;
 import static graphql.Directives.SkipDirective;
-import static graphql.language.NodeUtil.directivesByName;
 
```

Comment:
Create a new method `NodeUtil.directiveByName(String directiveName)` instead of introducing for-loops everywhere.
Similarly with arguments.

Reply:
Thanks for you review, the reason that not implemented by util method is that there is ’reture‘ in for loop. invoking `NodeUtil.directiveByName(List<Directive>,directiveName)` and then determine whether the return value is null will effect the readability of method.

Reply:
This will also affect performance

Reply:
> Create a new method NodeUtil.directiveByName(String directiveName) instead of introducing for-loops everywhere.
> Similarly with arguments.

THIS!

Reply:
Thanks for you review and explanation, the minor gc problem that I worried about would not happen.

As you said, the number of directives on an element will not too much, as well as the number arguments of Directive. In most cases, the improvement of this change is very limited.